bridge/config.go

```diff
@@ -0,0 +1,16 @@
+package bridge
+
+type Config struct {
+	// Interface is the network interface name, e.g. bond0.3, or ens33.
+	Interface string `json:"interface" yaml:"interface"`
+	// Name is the bridge name.
+	Name string `json:"name" yaml:"name"`
+	// PrivateNetwork is the host's private network to block against, e.g.
+	// 10.0.4.0/24.
+	PrivateNetwork string `json:"privateNetwork" yaml:"privateNetwork"`
+
+	// DNS holds DNS configuration for the bridge.
+	DNS DNS `json:"dns" yaml:"dns"`
+	// NTP holds NTP configuration for the bridge.
+	NTP NTP `json:"ntp" yaml:"ntp"`
```

Comment:
These settings are from the bridge's network unit, so I thought they belong to the bridge config.